### ISSUE TITLE
[v3] aggregate & map-reduce support

### ISSUE THREAD

**Issue by TongChia** [vadimdemedes/mongorito#162]
Are there plans to increase aggregation support?

**Comment by sitexw** [comment]
+1

**Comment by vadimdemedes** [comment]
I'd love to see it as an external plugin. Would you be willing to implement it?

**Comment by sitexw** [comment]
Indeed, it is conceivable that I could create a plugin for the management of aggregation.
I think about that...
